CHANGES.rst

```diff
@@ -131,6 +131,11 @@ API Changes
 
 - ``astropy.utils``
 
+  - On systems that do not have ``pkg_resources`` non-numerical additions to
```

Comment:
That's not an API change, right? It seems more like a bugfix. Not sure though, I generally struggle with these classifications. 😄